media-libs/x265/files/x265-3.6-code-cleanup.patch

```diff
@@ -0,0 +1,140 @@
+From bca0b4b72977683794d441e57adc8c279ff4bfca Mon Sep 17 00:00:00 2001
```

Comment:
I hope these have been sent upstream?

Reply:
Not yet. There is more to fix. Our patch list is long already.

I'll put all our changes at https://github.com/negril/x265_git/ and then forward it once complete and I had time to read into what they want. The upstream is on bitbucket anyway.